prfromtransifex.py

```diff
@@ -208,12 +208,17 @@ def createNewPullRequest(g,
         debug("Changed files: %s", " ".join(os.linesep.split(changedfiles)))
     
         info("Things changed & force pushing")
-        debug(git["commit", "-m", pr_commit % {'mode': mode,
-                            'minpercent': minpercent,
-                            'langcount': len(files)}]())
-        
-        debug(git["push", "-f", "origin", wr_branch]())
+        commit_msg = pr_commit % {'mode': mode, 'minpercent': minpercent, 'langcount': len(files)}
+        debug(git["commit", "-m", commit_msg]())
+
     
+        if pr:
+            info("PR exists already, squashing changes to commit in PR")
+            debug(git["reset", "--soft", "HEAD~1"])
```

Comment:
Using the base branch over the relative `HEAD~1` would be preferable.

Reply:
How'd that work? I mean instead of using `HEAD` what would I have to use?

Also feel free to make and commit any changes you want to this PR. This was just a fast draft of mine, so it won't be ideal in any case... :)

Reply:
I just tried to apply but it was already outdated. :P

I didn't see/quickly find the context here in this file yet otherwise I would have been more specific. HEAD~1 as a relative reference is not robust against changes and issues. I suspect this bot checks out master/a branch based on master, so that would be the base branch then we can reference here.

Not sure if the reset is even correct now? I guess I'll have to take another look later about the context.

Reply:
Thanks for looking into this :)